DSharpPlus.CommandsNext/CommandsNextExtension.cs

```diff
@@ -235,6 +235,9 @@ private async Task HandleCommandsAsync(DiscordClient sender, MessageCreateEventA
             }
 
             _ = Task.Run(async () => await this.ExecuteCommandAsync(ctx).ConfigureAwait(false));
+
+            if (this.Config.SignalHandledOnCommandFound)
+                e.Handled = true;
```

Comment:
This could simply be set to `e.Handled = this.Config.SignalHandledOnCommandFound`

Reply:
Furthermore, given my comment on the documentation, this should actually be `= !this.Config.SignalHandledOnCommandFound`